public/js/highlight-worker.js

```diff
@@ -0,0 +1,7 @@
+// A very simple web worker for highlight.js
+// See: https://highlightjs.org/usage/
+onmessage = function(event) {
+    importScripts('/vendor/plugins/highlight/highlight.pack.js');
```

Comment:
It must be possible to put importScripts outside of onmessage.

Reply:
And you will have to follow codacy (even if it is stupid) to be able to be merge by maintainers. Maybe if you add 
'use strict'; at start of file it may also not trigger for simple ' string

Reply:
Thanks @sapk - I've moved the import out in https://github.com/go-gitea/gitea/pull/3591/commits/7c2cc021ad6cd2091a3436c35ff2f55e4289a8ab - it still works fine in my testing, so :+1:  I've also added a `use strict` and will see how it goes, but I'll get codacy to pass somehow if not.